New Feature: Add ability to stash/unstash commands in interactive mode.

pohnean opened the pull request:
It's currently very cumbersome to retrieve successful commands performed in Interactive Mode. You often have to copy and paste commands line by line, and the commands do not have a trailing semi-colon.

This PR improves the interactive shell by adding 4 new keyboard shortcuts to allow the user to quickly stash and unstash the last run command.

1. Pressing F5 stashes the last run command.
2. Pressing F8 displays all the stashed commands with trailing semi-colons, so that it can be quickly pasted back into your IDE.
3. Pressing F10 clears the stashed commands.
4. Pressing F6 toggles 'auto-stashing', which will automatically stash a command if it successfully executes.

The following screenshot illustrates how it works:
![image](https://user-images.githubusercontent.com/846343/70369505-896e9000-18f5-11ea-8f31-76d6846a859c.png)

I have not written any test cases for this because I'm not sure about to go about writing test cases. Would appreciate if maintainers or other contributors help add it in.

Naktibalda commented on src/Codeception/Lib/Actor/Shared/Pause.php:
```diff
@@ -43,7 +119,11 @@ public function pause()
                         codecept_debug($result);
                     }
-                    codecept_debug('>> Result saved to $result variable, you can use it in next commands');
+                    codecept_debug('>> Result saved to $result variable, you can usZZe it in next commands');
```
Please remove `ZZ` from `usZZe`.

DavertMik replied:
```suggestion
                    codecept_debug('>> Result saved to $result variable, you can use it in next commands');
```

pohnean commented:
Okay, I've studied the code from CodeceptJS and added a similar feature that will save stashed commands automatically to `_output/stashed-commands`
Also removed the `ZZ` from `usZZe`.

DavertMik commented:
Awesome, thank you very much @pohnean
